bin/install-version

```diff
@@ -17,12 +17,15 @@ if [ ! "$#" -eq 1 ]; then
 	exit 1
 fi
 
+INSTALL_LOCATION=$HOME/.minimesos/bin
```

Comment:
It's a common pattern in *nix systems to store configs in dot prefix directories under $HOME. 
(nvm)[https://github.com/creationix/nvm] does it, and many more. So users won't find it strange or unusual.
I think install into current directory is more confusing, because you'd copy and paste the install command into the first terminal window available and pwd might be something totally random.